tests/Serilog.Sinks.RabbitMQ.Tests/Serilog.Sinks.RabbitMQ.Tests.csproj

```diff
@@ -33,6 +25,11 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Serilog.Sinks.RabbitMQ\Serilog.Sinks.RabbitMQ.csproj" />
+    <Using Include="NSubstitute" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net472' ">
+    <Using Remove="Newtonsoft.Json.Linq" />
```

Comment:
By the way I would not be so fond of making usings implicit as much usings as you can because of such conditions.